Chloe-Lin/Chloe-Lin.md

```diff
@@ -0,0 +1,14 @@
+---
+id: "6ed524e2-8933-eb11-a813-000d3a79983b&viewType=1039"
+
```

Comment:
any qualifications to be added here?

Reply:
No I don't have any qualifications for now 😢